router/src/server.rs

```diff
@@ -1352,6 +1359,192 @@ async fn metrics(prom_handle: Extension<PrometheusHandle>) -> String {
     prom_handle.render()
 }
 
+#[cfg(feature = "kserve")]
+#[utoipa::path(
+    post,
+    tag = "Text Generation Inference",
+    path = "/v2/health/live",
+    responses(
+        (status = 200, description = "Service is live", body = LiveReponse),
+        (status = 404, description = "Service not found", body = ErrorResponse,
+            example = json!({"error": "No response"}))
+    )
+)]
+async fn kserve_health_live() -> Result<Response, (StatusCode, Json<ErrorResponse>)> {
```

Comment:
Can we put all of these into the kserve file (the whole idea is to liimt the amount of conditional compilation flags).